test/extensions/filters/http/common/fuzz/filter_fuzz_test.cc

```diff
@@ -48,6 +48,7 @@ DEFINE_PROTO_FUZZER(const test::extensions::filters::http::FilterFuzzTestCase& i
   try {
     // Catch invalid header characters.
     TestUtility::validate(input);
+    ENVOY_LOG_MISC(info, "{}", input.config().DebugString());
```

Comment:
did you intend to leave this in?

Reply:
Thanks for catching, will create a debug log. I add it almost always when debugging this test.